src/main/java/com/amazon/ion/impl/LocalSymbolTableImports.java

```diff
@@ -242,13 +242,15 @@ SymbolTable getSystemSymbolTable()
      */
     SymbolTable[] getImportedTables()
     {
-        int count = myImports.length - 1; // we don't include system symtab
-        SymbolTable[] imports = new SymbolTable[count];
-        if (count > 0)
-        {
-            // defensive copy
-            System.arraycopy(myImports, 1, imports, 0, count);
-        }
+        // We have only the system symbol table, or we have none.
+        // None implies an empty system symbol table, as in Ion 1.1.
+        if (myImports.length == 1 || myImports.length == 0) return new SymbolTable[0];
```

Comment:
We might as well store `new SymbolTable[0]`. I can pick this up as a drive-by on one of my next PRs.